packages/http/src/validator/__tests__/functional.spec.ts

```diff
@@ -31,6 +31,87 @@ describe('HttpValidator', () => {
       });
     });
 
+    describe('deprecated keyword validation', () => {
+      const resource = {
+        id: "abc",
+        method: "get",
+        path: "/test",
+        responses: [
+          {
+            code: "200",
+            contents: [
+              {
+                mediaType: "application/json",
+                schema: {
+                  type: "object",
+                  properties: {
+                    prop: {
+                      type: "string",
+                      example: "abc"
+                    }
+                  },
+                  $schema: "http://json-schema.org/draft-04/schema#"
+                },
+              }
+            ]
+          }
+        ],
+        request: {
+          query: [
+            {
+              style: HttpParamStyles.Form,
+              deprecated: true,
+              name: "productId"
+            }
+          ],
+        },
+      };
+
+      it('returns warnings', () => {
+        expect(
+          validateInput({
+            // @ts-ignore
+            resource,
+            element: {
+              method: "get",
+              url: {
+                path: "/test",
+                query: {
+                  productId: "abc"
+                }
+              },
+            },
+          }),
+        ).toEqual([
+          {
+            code: "deprecated",
+            message: "Query param productId is deprecated",
+            path: [
+              "query",
+              "productId"
+            ],
+            severity: DiagnosticSeverity.Warning
+          }
+        ]);
+      });
+
+      it('does not return warnings', () => {
+        expect(
+          validateInput({
+            // @ts-ignore
+            resource,
+            element: {
+              method: "get",
+              url: {
+                path: "/test",
+                query: {}
+              },
+            },
+          }),
+        ).toEqual([]);
```

Comment:
```suggestion
        ).toHaveLenght(0);
```

Reply:
wanted to stay consistent within the file at least https://github.com/stoplightio/prism/blame/fcb5b1f1e1d71769d826e4a531988c8448321799/packages/http/src/validator/__tests__/functional.spec.ts#L29

Reply:
Fair.